.github/workflows/staging-pr-build.yml

```
@@ -1,30 +1 @@
-          distribution: temurin
-          java-version: 17
-          cache: gradle
-
-      - name: Grant execute permission for Gradle
-        run: chmod +x ./team1/gradlew || true
-
-      - name: Build with Gradle (clean build)
-        working-directory: ./team1
-        run: ./gradlew clean build --no-daemon
-
+p
```

actionlint (1.7.11)

[error] "jobs" section is missing in workflow

(syntax-check)